.github/workflows/sonar.yml

```diff
@@ -0,0 +1,22 @@
+name: SonarCloud
+
+on:
```

Comment:
@p-fruck mentioned last time, that it is also recommended to run scans on a regular basis to cover situations where new "detections rules" have been deployed, but there is no active development ongoing (and therefore now pr) in this repo.